### PR TITLE
LLVM 16

### PR DESCRIPTION
**Pull request by viktormalik**
This is a follow-up PR for #2528. It includes a fix for the CMake issue mentioned in https://github.com/iovisor/bpftrace/pull/2528#issuecomment-1487614582.

### CODE REVIEW

**Comment by viktormalik** [comment]
Once this is merged, we can do a release. @fbs will you handle it?

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,6 +1,10 @@
 cmake_minimum_required(VERSION 3.13.0)
 project(bpftrace)
 
+if(${CMAKE_VERSION} VERSION_EQUAL 3.3.0 OR ${CMAKE_VERSION} VERSION_GREATER 3.3.0)
```
Comment (ajor) [inline CMakeLists.txt:4]: We set the minimum required CMake version to 3.13 above, so I don't think this check is needed?

**Review by ajor — APPROVED** [review]

File: CMakeLists.txt (hunk 1 above)
Comment (danobi) [reply CMakeLists.txt:4]: Yeah, agreed

Comment (viktormalik) [reply CMakeLists.txt]: Ah, of course, I just copied the thing from BCC.

**Comment by viktormalik** [comment]
Merged, we're now ready to do a release.

**Pull request merged**